modules/search/class.jetpack-search-widget-filters.php

```diff
@@ -134,11 +143,96 @@ function widget( $args, $instance ) {
 		 */
 		do_action( 'jetpack_search_render_filters_widget_title', esc_html( $title ), $args['before_title'], $args['after_title'] );
 
+		// we need to dynamically inject the sort field into the search box when the search box is enabled, and display
+		// it separately when it's not.
 		if ( ! empty( $instance['search_box_enabled'] ) ) {
+			echo '<div class="jetpack-search-form">';
```

Comment:
Do you think an additional wrapper here may break the styling in some themes? I'm thinking of custom styling by the theme for the main search box widget that we still want to get applied for our search box also.

Reply:
It _should_ be able to withstand just being shoved into a div. I'm happy to remove this wrapper though - it's just there to make finding the correct form element easier.